core/testdata/getentryresponse.json

```diff
@@ -0,0 +1,2080 @@
+[
+	{
+		"Desc": "empty_alice",
+		"AppID": "app",
+		"UserID": "alice",
+		"Resp": {
+			"vrf_proof": "lhYlSe16x+JuLlfNjYgVCWGWs/WwzzaSRyjjCtL0IiBRQA/5SzU3buUY9Wy/fd0SFXqyRGteGIVRpoH/Yqw8TQSu+P2bNIFEUs3aiuNvY3HzppTZW2XMUJzlqTNUygh04vkDJokP3vnbgf/QymYIclb5GdAcv0dZyRxJYZvmLLek",
+			"leaf_proof": {
+				"leaf": {
+					"index": "S83qp3gG/wcb7OyR9YvaXAERw0b/v2QlW12ms7XFsuk="
+				},
+				"inclusion": [
+					"",
+					"",
```

Comment:
Woah, all the inclusion proofs in here seem to have loads of empty strings in them... is this right??

Reply:
This is correct. The test maps are mostly empty or have only one item in them. 
- Maps of size <2 will not have any non-empty leaves
- maps of size 2 <= 4 will have about 1 non-empty leaf.

Reply:
Ohhhh right, it's just clicked that this is in map-land.  Okay this makes more sense.